EssentialsDiscord/src/main/resources/config.yml

```
@@ -136,6 +140,15 @@ message-types:
   kick: staff
   # Message sent when a player's mute state is changed on the Minecraft server.
   mute: staff
+  # Message sent when a player talks in local chat.
+  # use-essentials-events must be set to "true" for this to work.
+  local: primary
```

Users won't expect local chat to be relayed by default
```suggestion
  local: none
```